tantalus/services.py

```diff
@@ -0,0 +1,55 @@
+import tantalus.models
+from tantalus.utils import *
+
+def get_form_changes(curation_instance, changed_fields, request, form_data, original_data, user_operation):
```

Comment:
with these changes it seems you can only modify a curation through the front end.  api access may be nice at some point for automation